tests/coalaTest.py

```diff
@@ -179,3 +179,24 @@ def test_logged_error_causes_non_zero_exitcode(self):
             )
 
             assert exitcode == 1
+
+    def test_coala_no_color(self):
+        with bear_test_module(), \
+                prepare_file(['#fixme'], None) as (lines, filename):
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala')
+            errors = filter(bool, stderr.split('\n'))
+            # Every error message must start with characters
+            # used for coloring.
+            for err in errors:
+                self.assertNotRegex(err, '^\[WARNING\]')
+            self.assertEqual(
+                retval, 0, 'coala must return zero when there are no errors')
```

Comment:
could you move the color testing into an own testing function `test_coala_color`?
(Hm maybe we should call then both functions `test_coala_log_with_colors` and `test_coala_log_without_colors` as `test_coala_color` sounds kind of bad :D)